Makefile

```diff
@@ -147,12 +147,16 @@ verify: check-tox ## Run tox -e fmt,lint,spellcheck against code
 spellcheck-sort: .spellcheck-en-custom.txt
 	sort -d -o $< $<
 
-.PHONY: man
-man: check-tox
+.PHONY: docs
+docs: check-tox  ## Generate Sphinx docs and man pages
 	tox -e docs
 
-.PHONY: docs
-docs: man ## Run tox -e docs against code
+.PHONY: man
+man: docs
+
+.PHONY: sphinx-apidoc
+sphinx-apidoc:
+	sphinx-apidoc --no-toc -o docs/api src/instructlab src/instructlab/llamacpp/llamacpp_convert_to_gguf.py
```

Comment:
This assumes `sphinx-apidoc` is installed. Can we first check it's present?

Reply:
I have an even better idea: `tox -e docs exec -- sphinx-apidoc ...`